content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -339,7 +338,7 @@ In the CLI, the access modes are abbreviated to:
 * ROX - ReadOnlyMany
 * RWX - ReadWriteMany
 
-> __Important!__ A volume can only be mounted using one access mode at a time, even if it supports many.  For example, a GCEPersistentDisk can be mounted as ReadWriteOnce by a single node or ReadOnlyMany by many nodes, but not at the same time.
+> __Important!__ A volume can only be mounted using one access mode at a time, even if it supports many.  For example, a GCEPersistentDisk can be mounted as ReadWriteOnce by a single node or ReadOnlyMany by many nodes, but not at the same time. Besides, it will be limited once a volume has been mounted on a single node with `ReadWriteOnce`, and pod which using this volume will be [checked](https://kubernetes.io/docs/concepts/scheduling/kube-scheduler/#filtering) whether it fit the volume mount in scheduling stage.
```

Comment:
If you're rewriting this, consider using a `{{< caution >}}` shortcode.

Reply:
Okay, I'll have a try.